### ISSUE TITLE
Problem about cv function in python wrapper[random_seed and auc score performance]

### ISSUE THREAD

**Issue by vatn** [catboost/catboost#73]
Hello, I'm now trying to test catboost with titanic dataset in kaggle.
Here is my code using cross validation function cv.
I always fix the other parameter values and just change random_seed each time:
`
#just for testing
from sklearn.model_selection import ParameterGrid
param_grid = {'learning_rate': [0.1], 
              'iterations': [5000],
              'l2_leaf_reg':[0.01],#
              'rsm':[0.8],#
              'border_count':[8],
              'fold_permutation_block_size':[1],
              'depth':[8],#6
              'gradient_iterations':[4],
              'bagging_temperature':[1],
              'loss_function': ['Logloss'] ,
              'eval_metric':['AUC'] ,
              'custom_loss':['Accuracy'],
              'random_seed':[10000] ###change random seed here
             }

param = list(ParameterGrid(param_grid))
nb_iteration = len(param)
scores = cv(param[0], pool,enable_early_stopping = False,fold_count = 4, random_seed= 2017,shuffle=True)`

each time i change random_seed, I output several cv scores and then I have the plot as following:
![untitled](https://user-images.githubusercontent.com/13531894/28833045-8658bd48-76de-11e7-8900-c1d99709d3ab.png)
[where y is auc score, x is the nb of iterations]

The auc scores output by 'cv' function change a lot with the change of random_seed.
I'm confused, can you help me to better understand this situation?
Thank you so much!

**Comment by annaveronika** [comment]
The reason for this result is probably that there is very small amount of data and target is very noisy, so when you change random_seed partition of data is changed and it changes result a lot. If you look on AUC_test_stddev you will see that it is very high.
If you will change random_seed of the algorithm, not of the partition, resulting auc will change not so much.
